pages/common/rg.md

```diff
@@ -0,0 +1,19 @@
+# ripgrep
+
+> A fast commandline search tool.
+
+- Recursively search the current directory for a regex pattern:
+
+`rg {{pattern}}`
+
+- Search in all .gitignored and hidden files:
+
+`rg -uu {{pattern}}`
+
+- Search only in html or css files for a pattern:
+
+`rg -thtml -tcss {{pattern}}`
+
+- Only search for a pattern in files matching a glob (e.g., 'README.*'):
```

Comment:
Please wrap the pattern in backticks, to signify a code sample, and to avoid markdown processors to choke on the asterisk.